src/TokenRequestFactory.js

```diff
@@ -1,7 +1,11 @@
 /* @flow */
 
+import { stringify } from 'query-string';
+
 import type { RequestFactory } from './types';
 
+require('fetch-everywhere');
```

Comment:
Curious: why `require` instead of `import`?

Reply:
Because `fetch-everywhere` executes side-effects by adding `global`s when requiring it: https://github.com/lucasfeliciano/fetch-everywhere/blob/b472faedba57cff3d5a6d1f2183812ab47a9f387/fetch-npm-node.js#L11-L16

Using `import` here would/could be tree-shaken or lazily-required, meaning we could execute our code before the side-effect occurred.

tl;dr: side-effects

Reply:
![](https://i.gifer.com/QyS7.gif)

Thanks!